modules/@apostrophecms/command-menu/index.js

```diff
@@ -16,8 +16,8 @@ module.exports = {
 
           const composed = pipe(self.composeCommand, self.composeRemove, self.composeGroup)({ rawCommands: self.rawCommands });
           try {
-            Object.entries(composed.command).every(([ name, command ]) => self.validateCommand({ name, command }));
-            Object.entries(composed.group).every(([ name, group ]) => self.validateGroup({ name, group }));
+            Object.entries(composed.command).some(([ name, command ]) => self.validateCommand({ name, command }));
```

Comment:
`some` stops on the first invocation that returns `true`, and `validateCommand` always does that, so I'm pretty sure it is wrong to use `some`.

As near as I can tell you don't care what the return value is, you're just stopping with an exception on any assert failures. That's fine, but don't return a value and use `some` if what you're really doing is iterating over everything unless an exception is thrown; it's confusing for later maintenance. Just use `forEach` and rely on the asserts as you have written them, unless I have misunderstood.